spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/interpreter/fixtures/MultipleMappingFixture.scala

```diff
@@ -90,7 +90,8 @@ trait MultipleMappingFixture extends BeforeAndAfterAll with SparkTestBase {
     val value = "../examples/data/e2e_tests/data/hdfs/std/mappingConformanceRuleData/2020/03/23/v1/_INFO"
     val infoFileContents: String = fromFile(value, "UTF-8").mkString
 
-    FileUtils.writeStringToFile(new File(s"${getRawDataPath(mappingDS.hdfsPath)}/_INFO"), infoFileContents, Charset.defaultCharset())
+    FileUtils.writeStringToFile(new File(s"${getRawDataPath(mappingDS.hdfsPath)}/_INFO"), infoFileContents,
```

Comment:
1. Why append true?
2. `Charset.defaultCharset` is being used in `NestedStructsFixture.scala` and `TradeConformance.scala`. I would like us to use the same across the board.

Reply:
I remember it didn't use to work without append true with the newer versions

Reply:
If the boolean param is needed, add naming to it please.
Also if touching it, there's a hint for `fromFile` to not be closed. Usually `FileReader` provides a safe way.

Reply:
I tried, `append = true` is not needed. Actually thinking about it, I have a suspicion it would cause problems on test reruns.
And **do name boolean parameters** when sending literals there.

Reply:
This file is not currently used anywhere, because its use is in `HyperConformanceMappingIntegrationSuite`. I would propose to move this discussion to #1745